docs/javascript-configuration.md

```diff
@@ -4,6 +4,67 @@ title: Scope Javascript Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified using environment variables or a configuration file `scope.yml` in the source root of the project.
+
+## Disabling Scope instrumentation
+
+Scope auto instruments your code and tests by default. You can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=false
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  scope:
+  instrumentation:
+    enabled: false
+
+```
+
+## Changing service name
+
+You can specify the name of the service when sending data to Scope. Scope uses `default` as fallback.
+
+### Using Environment Variables
+
+```sh
+SCOPE_SERVICE="service-name"
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  service: 'service name'
+```
+
+## Change testing mode
+
+This variable indicates whether the agent is running tests (`true`) or it is being used for runtime instrumentation (`false`).
+
+### Using Environment Variables
+
+```sh
+SCOPE_TESTING_MODE=false
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  mode:
+    testing: false
+  
+```
```

Comment:
this is not yet implemented in javascript. Let's remove the section

Reply:
Deleted